modules/images-other-jenkins-config-kubernetes.adoc

```diff
@@ -9,14 +9,18 @@
 The {product-title} Jenkins image includes the pre-installed link:https://wiki.jenkins-ci.org/display/JENKINS/Kubernetes+Plugin[Kubernetes plug-in] that allows Jenkins agents to be dynamically provisioned on multiple container hosts using Kubernetes and {product-title}.
 
 To use the Kubernetes plug-in, {product-title} provides images that are suitable for use as Jenkins agents, including the Base, Maven, and Node.js images.
+// UPDATE?
```

Comment:
we need to add a link about the deprecation section telling that use of specific images are deprecated and we support the other way.